src/index.html

```diff
@@ -21,7 +24,413 @@
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__rating-container">
+          <span class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </span>
+
+          <span class="card__reviews-count">Reviews: 5</span>
+        </div>
+
+        <div class="card__price-container">
+          <span class="card__price-label">Price:</span>
+
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy-link"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="iMac image"
+        />
+
+        <h2 class="card__product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__rating-container">
+          <span class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </span>
+
+          <span class="card__reviews-count">Reviews: 5</span>
+        </div>
+
+        <div class="card__price-container">
+          <span class="card__price-label">Price:</span>
+
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy-link"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="iMac image"
+        />
+
+        <h2 class="card__product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__rating-container">
+          <span class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </span>
+
+          <span class="card__reviews-count">Reviews: 5</span>
+        </div>
+
+        <div class="card__price-container">
+          <span class="card__price-label">Price:</span>
+
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy-link"
+        >
+          Buy
+        </a>
```

Comment:
Repeated use of the 'card' class suggests that these should be components. Consider using a '<section>' tag with a class 'catalog__cards' to contain all your 'card' components for better semantic structure.